src/components/common/buttons/CopyButton.js

```diff
@@ -48,7 +55,13 @@ const CopyButton = ({ toCopy, children, onPress, onPressDone, iconColor, without
     }
     default: {
       return (
-        <CustomButton data-gdtype={'copybutton'} mode={mode} onPress={onPressHandler} {...props}>
+        <CustomButton
+          data-gdtype={'copybutton'}
+          disabled={!hasClipboardAccess}
```

Comment:
it should not be disabled. it should be enabled, when you press on it he sees dialog if no permissions